### DISCUSSION TITLE
Publish to F-Droid?

### DISCUSSION THREAD

**Discussion by MiepHD** [zixpo/candybar#127] [Q&A] [answered]
I want to publish my icon pack on F-Droid. To do this I need to remove the BillingClient dependency, (How) can I do this?
Thanks in advance

**Comment by Donnnno** [comment, marked as answer]
Hey! If you're interested in publishing your icon pack to F-Droid, you should check out Candybar FOSS here: https://github.com/Donnnno/candybar-foss 

It works basically the same as regular Candybar, you just have to change this line:
`    implementation 'com.github.zixpo:candybar:3.16.2'`
to this in your build.gradle:
`    implementation 'com.github.Donnnno:candybar-foss:3.16.2'`
